doc/source/ray-contribute/development.rst

```diff
@@ -143,6 +143,8 @@ To build Ray on Ubuntu, run the following commands:
   nvm install 14
   nvm use 14
 
+.. note::
+  We recommend using `bazelisk` for building. If you prefer to use `bazel`, ensure it is version `5.4.1`.
```

Comment:
Can you explicitly call out that `bazelisk` is what `install-bazel.sh` installs?

Reply:
done